python/debugging/PartialPathsFromSource.ql

```diff
@@ -0,0 +1,42 @@
+/**
+ * @name Partial Path Query
+ * @description Partial Path Query
+ * @kind path-problem
+ * @problem.severity error
+ * @security-severity 10.0
+ * @precision high
+ * @sub-severity high
+ * @id py/debugging/partial-sql-injection
+ * @tags debugging
+ */
+
+import python
+import semmle.python.dataflow.new.DataFlow
+import semmle.python.dataflow.new.TaintTracking
+import semmle.python.Concepts
+import semmle.python.dataflow.new.RemoteFlowSources
+import semmle.python.dataflow.new.BarrierGuards
+import DataFlow::PartialPathGraph
+import semmle.python.ApiGraphs
+// Helpers
+import github.Helpers
+import github.LocalSources
+// Bring in the CommandInjection config
+private import semmle.python.security.dataflow.CommandInjectionCustomizations
+
+// Parcial Graph
```

Comment:
Can you fix the typo please? Partial, not parcial

Reply:
Fixed typos